Marlin/configuration_store.cpp

```diff
@@ -36,10 +36,10 @@
  *
  */
 
-#define EEPROM_VERSION "V23"
+#define EEPROM_VERSION "V24"
```

Comment:
As i would understand it, the default is for using on Terminal.

If a host needs faster or slower response it should set while connecting and reset on disconnect.

So @repetier and @foosel could have an own (even stored) parameter for this and tune it for their side of the protocol.